apps/admin_audit/lib/AuditLogger.php

```diff
@@ -50,39 +48,39 @@ public function __construct(ILogFactory $logFactory, IConfig $config) {
 		$this->parentLogger = $logFactory->getCustomPsrLogger($logFile, $auditType, $auditTag);
 	}
 
-	public function emergency($message, array $context = array()) {
+	public function emergency($message, array $context = array()): void {
```

Comment:
I would prefer the new way:
```suggestion
	public function emergency($message, array $context = []): void {
```
Same for next functions